src/features/TodoForm.jsx

```diff
@@ -0,0 +1,31 @@
+import { useRef } from 'react';
+import { useState } from 'react';
+import TextInputWithLabel from '../shared/TextInputWithLabel';
+
+function TodoForm({ onAddTodo }) {
+  const todoTitleInput = useRef('');
+  const [workingTodoTitle, setWorkingTodoTitle] = useState('');
+
+  function handleAddTodo(title) {
+    title.preventDefault();
+    onAddTodo({ 'id': Date.now(), 'title': workingTodoTitle, 'isCompleted': false });
+    setWorkingTodoTitle('');
+    todoTitleInput.current.focus();
+  }
+
+  return (
+    <>
+      <form onSubmit={handleAddTodo}>
+        <TextInputWithLabel
+          elementId="todoTitle"
```

Comment:
Conventionally, when passing in a prop argument to a component you want to user `{}`